e2e-tests/commands/insurance/your-buyer/complete-credit-insurance-cover-form.js

```diff
@@ -0,0 +1,26 @@
+import { yesRadioInput, noRadioInput, field } from '../../../pages/shared';
+import { YOUR_BUYER as FIELD_IDS } from '../../../constants/field-ids/insurance/your-buyer';
+
+const {
+  PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER,
+} = FIELD_IDS;
+
+/**
+ * completeCreditInsuranceCoverForm
+ * completes credit insurance cover form
+ * if hasHadCreditInsuranceCover, then selects yes radio and fills in conditional text area
+ * if not hasHadCreditInsuranceCover then selects no radio
+ * @param {Boolean} hasHadCreditInsuranceCover - should select yes or no radio
+ */
+const completeCreditInsuranceCoverForm = ({ hasHadCreditInsuranceCover = false }) => {
+  const fieldId = PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER;
+
+  if (hasHadCreditInsuranceCover) {
+    yesRadioInput().click();
+    cy.keyboardInput(field(fieldId).textarea(), 'test');
```

Comment:
Can the value come from mock application, something more meaningful e.g "Mock cover with buyer"